app/Server/src/main/java/Servlets/SkierServlet.java

```diff
@@ -181,9 +188,16 @@ protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws Se
                 resp.setStatus(HttpServletResponse.SC_BAD_REQUEST);
```

Comment:
I highly recommend we put a zero (SkierVertical.addResortItem([]) or at least return 200 status so that client treat it as a failure